src/Groups/cosets.jl

```diff
@@ -258,12 +288,12 @@ Permutation group of degree 4 and order 24
 julia> H = symmetric_group(3)
 Permutation group of degree 3 and order 6
 
-julia> left_cosets(G,H)
+julia> left_cosets(G, H)
```

Comment:
Now `left_cosets` behaves quite different from `right_cosets`... We should either warn about this; or else turn that into a G-set, too (I guess by letting $g \in G$ act via $(aU).g := g^{-1}aU$ ?)

Reply:
Do we want to support left cosets at all?
In GAP, there is no function `LeftTransversal`. The GAP function `LeftCosets` belongs to the utils package (without documentation, the documentation belongs to the groupoids package). I would recommend not to use left cosets.